vms/volunteer/views.py

```diff
@@ -64,6 +65,20 @@ def delete_resume(request, volunteer_id):
     else:
         return HttpResponse(status=403)
 
+def get_job_list(request):
+    result = []
+    user = request.user
+    volunteer_id = user.volunteer.id
+    event_name = request.GET.get('event_name', None)
+    event = get_event_by_name(event_name)
+    event_id = event.id
+    jobs = get_jobs_by_event_id(event_id)
+    job_list = get_signed_up_jobs_for_volunteer(volunteer_id)
+    for j in jobs:
+        if(any(j.name == job for job in job_list)):

```

Comment:
if it has an id, please use comparison with id please. 

also, dont use single char var names like `j`